docs/changelog/v1.16.0.md

```diff
@@ -0,0 +1,13 @@
+# v1.16.0
+
+## Changes
+
+### Operator
+
+Ensure we call exclude only on missing processes  bug  Something isn't working technical-debt [#1550](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1550)
```

Comment:
```suggestion
Ensure we call exclude only on missing processes [#1550](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1550)
```

was it intended to include those labels?

Reply:
there is no script for this project. this is manually created.